tests/unit/test-cmd/test.build.js

```diff
@@ -72,6 +72,40 @@ describe('build', () => {
     });
   });
 
+  it('throws an error if the filename manifest value is not a string', () => {
```

Comment:
Nit, `throws on missing manifest properties in filename template`? or something like that.